build/linux/installer/scripts/tomlparser-agent-config.rb

```diff
@@ -164,6 +204,16 @@ def populateSettingValuesFromConfigMap(parsedConfig)
   file.write("export HPA_CHUNK_SIZE=#{@hpaChunkSize}\n")
   file.write("export PODS_EMIT_STREAM_BATCH_SIZE=#{@podsEmitStreamBatchSize}\n")
   file.write("export NODES_EMIT_STREAM_BATCH_SIZE=#{@nodesEmitStreamBatchSize}\n")
+  # fbit settings
+  if @fbitFlushInterval > 0
+    file.write("export FBIT_SERVICE_FLUSH_INTERVAL=#{@fbitFlushInterval}\n")
```

Comment:
we support only megs & secs. so these should have explicit units.

Reply:
> we support only megs & secs. so these should have explicit units.

For chunksize(s), "m" being appened by https://github.com/microsoft/Docker-Provider/blob/ci_dev/build/common/installer/scripts/td-agent-bit-conf-customizer.rb and for flush interval, there is no unit for the interval unit suffix. Flush interval value supported in seconds. I kept the same contract as we support for hyrbid via environment variables.

Basically, here in this file, we are parsing configmap and setting the env vars, which will be inturn parsed by td-agent-bit-conf-customizer.rb code and setting the td-agent conf file.

Reply:
External setting names must have units. How will people know what FBIT_TAIL_BUFFER_MAX_SIZE=1 mean ? 1k ? 1g? 1m?

Reply:
> External setting names must have units. How will people know what FBIT_TAIL_BUFFER_MAX_SIZE=1 mean ? 1k ? 1g? 1m?

> agree, having the units suffix in name of the config key is self explanatory.  For example FBIT_TAIL_BUFFER_MAX_SIZE_MB but changing this will not keep the same naming contract what we support in HELM chart.   In configmap, these will be 

FBIT_SERVICE_FLUSH_INTERVAL_SECS
FBIT_TAIL_BUFFER_CHUNK_SIZE_MB
FBIT_TAIL_BUFFER_MAX_SIZE_MB

but where as in helm or yaml will be  without unit suffix. since changing these will break the existing customers

FBIT_SERVICE_FLUSH_INTERVAL
FBIT_TAIL_BUFFER_CHUNK_SIZE
FBIT_TAIL_BUFFER_MAX_SIZE

Reply:
Fixed. Please have a look a it.